src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -713,6 +867,38 @@ public boolean deleteMetadata(String pid, String formatId) throws IllegalArgumen
         return true;
     }
 
+    /**
+     * Overload method for retrieveMetadata with default metadata namespace
+     */
+    public boolean deleteMetadata(String pid) throws IllegalArgumentException,
+        FileNotFoundException, IOException, NoSuchAlgorithmException {
+        logFileHashStore.debug(
+            "FileHashStore.deleteMetadata - Called to delete metadata for pid: " + pid
+        );
+        // Validate input parameters
+        isObjectNull(pid, "pid", "deleteMetadata");
+        isStringEmpty(pid, "pid", "deleteMetadata");
+
+        // Get permanent address of the pid by calculating its sha-256 hex digest
+        Path metadataCidPath = getRealPath(pid, "metadata", METADATA_NAMESPACE);
+
+        // Check to see if object exists
+        if (!Files.exists(metadataCidPath)) {
+            String errMsg = "FileHashStore.deleteMetadata - File does not exist for pid: " + pid
+                + " with metadata address: " + metadataCidPath;
+            logFileHashStore.warn(errMsg);
+            throw new FileNotFoundException(errMsg);
+        }
+
+        // Proceed to delete
+        deleteObjectAndParentDirectories(metadataCidPath, pid, "deleteMetadata");
+        logFileHashStore.info(
+            "FileHashStore.deleteMetadata - File deleted for: " + pid + " with metadata address: "
+                + metadataCidPath
+        );
+        return true;
```

Comment:
why return a boolean, if it will always be `true`? This may mislead devs into writing client code that needs logic to account for it returning false (which never happens), in addition to catching and responding to exceptions.

Reply:
Ah - got it I've removed it. Thank you for explaining - I thought that the calling app might want a boolean to confirm that it's actually been deleted... but can see why this may become problematic.

Reply:
I guess if you wanted to do that, you could instead catch the exceptions and translate them into log entries and a boolean `false`. What's confusing is the mixture of "returns true if it succeeds, but throws an exception if it fails"